packages/console/src/components/Steps/components/StepItem/index.module.scss

```diff
@@ -0,0 +1,32 @@
+@use '@/scss/underscore' as _;
+
+.step {
```

Comment:
nit: do we need to define the step class name here? Looks like no styles are applied

Reply:
It is used in the `StepItem/index.tsx`.